qa/suites/upgrade/clients-upgrades-jewel-nautilus/jewel-client-x/rbd/supported/ubuntu_18.04.yaml

```diff
@@ -0,0 +1 @@
+../../../../../../distros/all/ubuntu_18.04.yaml
```

Comment:
Nit: not supported

Reply:
@dillaman I don't think it matters as we use filters anyway, agree?

Reply:
Just seems cleaner to yank it since it's not supported.

Reply:
@dillaman done also renamed the suite to be consistent